### ISSUE TITLE
Tensor shape not match when modifying y_hat if pred_len is odd

### ISSUE THREAD

**Issue by Newaij0** [SJTU-Quant/LIFT#5]
## Problem

When modifying y_hat, a shape mismatch error will occur. I notice that you use  `y_hat = y_hat + torch.fft.irfft(self.mix_layer(torch.cat([seq_shifted_f.sum(2), seq_diff_f.sum(2), y_hat_f], -1)))` to modify the final result in the **LIFT.py** file, but there will be a problem if the `pred_len` is an odd number.

According to the official website of torch.fft.irfft <https://pytorch.org/docs/stable/generated/torch.fft.irfft.html> if the output dim is not specified the output will not round-trip properly.

## Soution 
Maybe add a specification of the output dim in the code.
`y_hat = y_hat + torch.fft.irfft(self.mix_layer(torch.cat([seq_shifted_f.sum(2), seq_diff_f.sum(2), y_hat_f], -1)),self.pred_len)`

**Comment by MogicianXD** [comment]
Great! Thanks for your contribution!!😊 We have updated the code.
